### DISCUSSION TITLE
Unified video gen collaboration

### DISCUSSION THREAD

**Discussion by BinaryQuantumSoul** [volotat/SD-CN-Animation#47] [Ideas]
There are few different tools for generating videos with stable diffusion. It could be interesting that you all collaborate in order to have an unified powerful way of doing videos with SD. 

- https://github.com/s9roll7/ebsynth_utility
- https://github.com/deforum-art/sd-webui-deforum
- https://github.com/CiaraStrawberry/TemporalKit

**Comment by volotat** [comment]
Thoretically this sounds awesome, but to be honest I can harly imagine how this could be done. As I understand it all of these tools use quite different approach to achive its results. And most likely every developer has their own view on how this could be improved and move forward. Lastly, most likely all of our tools are doomed right at the moment when good enough open source text2video model became public.

**Reply by BinaryQuantumSoul** [reply]
Makes sense, thanks for the reply
